runtime/oti/j9nonbuilder.h

```diff
@@ -4521,6 +4521,7 @@ typedef struct J9InternalVMFunctions {
 	IDATA  ( *objectMonitorEnterBlocking)(struct J9VMThread *currentThread) ;
 	void  ( *fillJITVTableSlot)(struct J9VMThread *vmStruct, UDATA *currentSlot, struct J9Method *ramMethod) ;
 	IDATA  ( *findArgInVMArgs)(J9PortLibrary *portLibrary, struct J9VMInitArgs* j9vm_args, UDATA match, const char* optionName, const char* optionValue, UDATA doConsumeArgs) ;
+	IDATA  ( *findArgInVMArgsImpl)(J9PortLibrary *portLibrary, struct J9VMInitArgs* j9vm_args, UDATA match, const char* optionName, const char* optionValue, UDATA doConsumeArgs, UDATA doConsumeIgnoredArgs) ;
```

Comment:
this function isn't called from outside the j9vm lib, so it doesnt need to be in the internal function table

Reply:
still needs to be addressed